snapcraft/plugins/make.py

```diff
@@ -89,7 +89,7 @@ def get_build_properties(cls):
     def __init__(self, name, options, project):
         super().__init__(name, options, project)
 
-        if project.info.base not in ("core", "core16", "core18"):
+        if project.info.base not in ("none", "core", "core16", "core18", "core20"):
```

Comment:
none is still not a base, mind removing it?

Reply:
That depends on what should I use in the core20 snapcraft.yaml.

At the moment, I thought that core20 snap would use "base: none" in snapcraft.yaml and it does use make plugin.

Or should core20 snapcraft declare "base: core20" ?

it sounded like a self-reference to me, but after Gustavo voiced it, it didn't sound that odd.

Reply:
#2623 has the right information encoded to make things work.